addons/l10n_ar/__manifest__.py

```diff
@@ -2,8 +2,12 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Argentina - Accounting',
+<<<<<<< HEAD
     'icon': '/base/static/img/country_flags/ar.png',
     'version': "3.4",
+=======
+    'version': "3.5",
+>>>>>>> 197b180358e... temp
```

Comment:
Can you please stop changing the version? It is useless, most likely wrong, and it just creates conflicts.
@Bruno-Zanotti